### ISSUE TITLE
Support deleting zapps

### ISSUE THREAD

**Issue by puf** [invertase/zapp.run#101]
I accidentally forked a Zapp, and now can't delete it. I renamed the fork to make it clear to myself that I shouldn't work in that specific zap, but still risk opening it by accident. Allowing me to delete the zapp would be a better option here.

**Comment by Dawntraoz** [comment]
Same here, I will love to clean up my Zapps 🙋🏼‍♀️

Thank you for this amazing platform, by the way! If I can contribute somehow, I will be more than happy 💜

**Comment by rivella50** [comment]
Same here, the list of my Zapp projects connected to my Github account gets bigger and bigger. Therefore a delete function would be highly appreciated.

**Comment by ThatDoe** [comment]
Yes, I joined a discord channel to ask how this was possible. But apparently from the existence of this thread, it isn't possible yet. 

Thank you for giving me a platform to play with flutter when away from my computer in the first place! Cheers! 🍻
